prime-router/src/main/kotlin/cli/SettingCommands.kt

```diff
@@ -106,8 +106,9 @@ abstract class SettingCommand(
             .jsonBody(payload)
             .responseJson()
         return when (result) {
-            is Result.Failure ->
-                throw result.getException()
+            is Result.Failure -> {
+                throw Exception("Error on put of $settingName: ${String(response.data)}", result.getException())
+            }
             is Result.Success ->
```

Comment:
Added the API error message to the exception.

Reply:
Thanks!